node/src/neighborhood/gossip_acceptor.rs

```diff
@@ -3353,7 +3515,31 @@ mod tests {
             make_default_neighborhood_metadata(),
         );
 
-        assert_eq!(result, GossipAcceptanceResult::Accepted);
+        let mut root_node = dest_db.root().clone();
+        root_node.clear_half_neighbors();
+        root_node
+            .add_half_neighbor_key(src_node.public_key().clone())
+            .expect("TODO: panic message");
+        let gnr = GossipNodeRecord::from((
+            root_node.inner.clone(),
+            root_node.node_addr_opt(),
+            main_cryptde(),
+        ));
+        let debut_gossip = Gossip_0v1 {
+            node_records: vec![gnr],
+        };
+        assert_eq!(
+            result,
+            GossipAcceptanceResult::Reply(
+                debut_gossip,
+                src_node.public_key().clone(),
+                src_node
+                    .node_addr_opt()
+                    .as_ref()
+                    .expect("Debut gossip always has an IP")
+                    .clone(),
+            )
+        );
```

Comment:
Since these green lines are exactly what those in the previous test are, I'd like to suggest to create an outboard method containing it, called twice from a different test each time. 

The name could be like "make_expected_non_introduction_debut_response".